Documentation/ceph-cluster-crd.md

```diff
@@ -134,6 +134,7 @@ The following storage selection settings are specific to Ceph and do not apply t
 - `walSizeMB`:  The size in MB of a bluestore write ahead log (WAL). Include quotes around the size.
 - `journalSizeMB`:  The size in MB of a filestore journal. Include quotes around the size.
 - `osdsPerDevice`**: The number of OSDs to create on each device. High performance devices such as NVMe can handle running multiple OSDs. If desired, this can be overridden for each node and each device.
+- `encryptedDevice`: Encrypt OSD volumes using dmcrypt ("true" or "false").
```

Comment:
This must be moved below the `NOTE` block as the `osdsPerDevice` includes that block.

I would also like to see this added to the "full" cluster example yaml.

Reply:
- Instead of moving it below the `NOTE` block, I think we just need to add `**` to this property similar to the `osdsPerDevice`. 
- The `encryptedDevice` property could be mentioned in the next paragraph. 
- Agreed that adding an example line in the cluster.yaml would be good. LIkely it should be commented out by default.